test/unit/clickable-component.test.js

```diff
@@ -111,3 +111,32 @@ QUnit.test('handleClick should use handler from options', function(assert) {
   testClickableComponent.dispose();
   player.dispose();
 });
+
+QUnit.test('language change should localize its text', function(assert) {
+  assert.expect(2);
+
+  const player = TestHelpers.makePlayer({
+    languages: {
+      es: {
+        Play: 'Juego'
+      },
+      en: {
+        Play: 'Play'
+      }
+    }
+  });
+
+  const testClickableComponent = new ClickableComponent(player);
+
+  testClickableComponent.controlText_ = 'Play';
+  const el = testClickableComponent.createEl();
+
+  player.language('en');
+  assert.ok(el.innerHTML.match(/vjs-control-text"?[^<>]*>Play/));
```

Comment:
instead of string matching, we should use DOM methods to get the text of the control text.
For example:
```js
el.querySelector('.vjs-control-text').textContent
```
Then we can use `assert.equal` for greater visibility into what this check is doing.